src/bin/build.rs

```diff
@@ -43,6 +44,7 @@ Options:
     -h, --help                   Print this message
     -p SPEC, --package SPEC ...  Package to build
     --all                        Build all packages in the workspace
+    --exclude SPEC ...           Exclude packages from the build
```

Comment:
> Basically we need to switch to clap

Oh it is desired to switch to `clap`? It's good to hear that! Is there someone working on it already? If not, I'd love to do it.